packages/hagrid/hagrid/orchestra.py

```diff
@@ -364,6 +364,11 @@ def deploy_to_container(
     env = os.environ.copy()
     env["USER"] = USER
 
+    # stdlib
+    import sys
+
+    print(" ".join(commands), file=sys.stderr)
```

Comment:
is this debugging code?

Reply:
omg yes sorry about that, will remove it